src/components/nve-switch/nve-switch.component.ts

```diff
@@ -0,0 +1,148 @@
+import { customElement, property, query, state } from 'lit/decorators.js';
```

Comment:
du trenger å kjøre `npm run manifest` for å vise jsdoc beskrivelsen i markdown sider